labs/lab-1-html-and-css/mobilefirststyle.css

```diff
@@ -0,0 +1,206 @@
+*{
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 800;
+}
+
+@media (max-width: 600px){
+
+body {
+  background-color: #f9fafc;
+}
+
+p{
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 800;
+  color: #69696a;
+}
+
+nav a {
+  display: block;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 800;
+  margin: 10px;
+  position: relative;
+}
+
+a:link {
+  color: #69696a;
+  text-decoration: none;
+}
+
+.top-bar {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.top-header {
+  display: inline;
+}
+
+.main-paragraph {
+  border-left: 3px solid #69696a;
+  padding-left: 15px;
+  margin: 20px;
+}
+
+#scroll-down {
+  position: relative;
+  bottom: 0px;
+  margin: 20px;
+  background-color: black;
+  border-radius: 40px;
+  width: 95%;
+  padding: 10px 0px;
+  color: white;
+  font-size: 14px;
+  text-align: center;
+}
+
+.black-button {
+  background-color: black;
+  border-radius: 40px;
+  width: 100px;
+  padding: 10px 0px;
+  color: white;
+  font-size: 14px;
+  text-align: center;
+  width: 90%;
+  margin: auto;
+
+  }
+}
+@media (min-width: 600px) {
+  body{
+    background-color: #f9fafc;
+  }
+
+  h1{
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 800;
+  }
+
+  h3{
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 800;
+  }
+
+  p{
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 800;
+    color: #69696a;
+  }
+
+  nav a {
+    display: inline;
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 800;
+    margin: 10px;
+    position: relative;
+  }
+
+  a:link {
+    color: #69696a;
+    text-decoration: none;
+  }
+
+  .top-bar {
+    display: inline;
+  }
+
+  .top-header {
+    display: inline;
+  }
+
+  .navigation {
+    float: right;
+  }
+
+  .main-content {
+    margin-top: 100px;
+    display: flex;
+    position: relative;
+  }
+
+  .main-header {
+    margin: 20px;
+  }
+
+  .main-paragraph {
+    border-left: 3px solid #69696a;
+    padding-left: 10px;
+    width: 50%;
+    margin: 20px;
+  }
+
+  .circles {
+    position: relative;
+    width: 50%;
+    height: 400px;
+  }
+
+  .circle-red {
+    position: absolute;
```

Comment:
You position these `absolute` here, which is what I'd expect, but then you position them `relative` in the other css file. Remember, ALL the css rules for a selector will apply unless they're in a media-query; and the last version of any given rule wins.

On the whole, I'd expect not to see this whole `@media (min-width: 600px)` block here at all; I'd expect to see all this in your other file, probably?

I'll talk about this more in class, it's clearly tripping several people up.